python/benchmarks/real_query_builder.py

```diff
@@ -0,0 +1,166 @@
+"""
+Copyright 2025 Man Group Operations Limited
+
+Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
+
+As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+"""
+
+import time
+import pandas as pd
+from arcticdb.util.environment_setup import SetupSingleLibrary, Storage
+from arcticdb.version_store.library import Library
+from arcticdb.version_store.processing import QueryBuilder
+
+from benchmarks.common import  generate_benchmark_df
+from benchmarks.local_query_builder import PARAMS_QUERY_BUILDER
+
+
+#region Setup classes
+
+class QueryBuilderFunctionsSettings(SetupSingleLibrary):
+    """
+    Responsible for setup special read only symbols generated based
+    on special dataframe for benchmarking queries
+    """
+
+    def generate_dataframe(self, rows:int, columns: int) -> pd.DataFrame:
+        """
+        Dataframe that will be used in read and write tests
+        """
+        st = time.time()
+        # NOTE: Use only setup environment logger!
+        self.logger().info("Dataframe generation started.")
+        df = generate_benchmark_df(rows)
+        self.logger().info(f"Dataframe {rows} rows generated for {time.time() - st} sec")
+        return df
+
+#endregion
+
+class AWSQueryBuilderFunctions:
+    """
+    This is same test as :LocalQueryBuilderFunctions:`LocalQueryBuilderFunctions`
```

Comment:
Is there really no way to share the code between them, for example by pulling out a subclass?